tests/dom/position.js

```diff
@@ -159,6 +159,31 @@ describe( 'getOptimalPosition()', () => {
 				name: 'left'
 			} );
 		} );
+
+		// https://github.com/ckeditor/ckeditor5-utils/issues/148
+		it( 'should return coordinates (#3)', () => {
+			const overflowedAncestor = getElement( {
+				top: 100,
+				left: 0,
+				bottom: 110,
+				right: 10,
+				width: 10,
+				height: 10
+			}, {
+				overflow: 'scroll'
```

Comment:
Looks like `overflow: scroll` is no longer needed by this test but OTOH it better explains this test case.